sdks/python/apache_beam/io/gcp/bigquery_read_it_test.py

```diff
@@ -259,6 +259,43 @@ def test_table_schema_retrieve_with_direct_read(self):
           ]))
 
 
+class ReadUsingReadGbqTests(BigQueryReadIntegrationTests):
```

Comment:
I'd suggest making a new `dataframe.io_it_test` for these, rather than adding them here. Will that work?